.neutrinorc.js

```diff
@@ -81,59 +69,42 @@ module.exports = {
             "react/jsx-curly-brace-presence": 0,
             "react/jsx-no-target-blank": 0,
             "object-curly-newline": 0,
-            "react/no-unused-state": 0
+            "react/no-unused-state": 0,
+            "jest/valid-expect": 0,
           }
         }
       }
     ],
+    [
+      '@neutrinojs/web', {
+        devtool: {
+          // Enable source-maps in production
+          production: 'source-map'
+        },
+        env: [
+          'BACKEND',
+        ],
+        // Read https://stackoverflow.com/a/36623117
+        // This is the key to making React Router work with neutrino
+        // Fix issue with nested routes e.g /index/garbage
+        publicPath: '/',
+      }
+    ],
     [
       '@neutrinojs/react',
       {
         html: {
           title: 'Firefox Health Dashboard',
           links: [
-            "https://fonts.googleapis.com/css?family=Fira+Sans:300,400,400i,500",
             {
               href: '/static/favicon.ico',
```

Comment:
This used to be copied to the output folder via the copy plugin being built into the web preset. Now it needs to be configured manually. See:
https://github.com/neutrinojs/neutrino/pull/814/files#diff-5e2fbcbeed183e062b9d4c1df42402f4L139